inc/packages/class-metadatadocument.php

```diff
@@ -42,6 +42,20 @@ class MetadataDocument {
 	 */
 	public $slug;
 
+	/**
+	 * File name.
+	 *
+	 * @var string
+	 */
+	public $file;
+
+	/**
+	 * Is package binary a release asset?
+	 *
+	 * @var bool
+	 */
+	public $release_asset;
```

Comment:
I'm curious whether we need this additional property, or if we should actually just be using the `$releases` property which contains an array of `ReleaseDocument` objects, which I assume means they're release assets.

Is there some way to use what's in that property to accomplish the goal here?

Reply:
There is no identifier of a release asset. Closest thing would be to parse download URI but that's not a good idea.

Reply:
GitHub release assets require an Accept header of application/octet-stream in order to download.

Reply:
I suppose I'm wondering this:

If we have a MetadataDocument and a ReleaseDocument, why can the MetadataDocument be considered a release asset? Shouldn't the only release assets be `ReleaseDocument` objects?

Reply:
A release asset is a type of package binary.